scripts/prepare-code-coverage-artifact.py

```diff
@@ -156,13 +172,12 @@ def get_commandline_args():
     apollo_binary_path = os.path.join(
         'build', 'tests', 'simpleKVBC', 'TesterReplica', 'skvbc_replica')
 
-    # default llvm-profdata and llvm-cov binary from docker image
-    profdata_exe = 'llvm-profdata'
-    cov_exe = 'llvm-cov'
-
     # Path to the directory containing the raw profiles
     profile_data_dir = os.path.join(args.apollo_build_path, 'codecoverage')
 
+    # path to llvm-profdata and llvm-cov binary
+    profdata_exe, cov_exe = get_llvm_tools(profile_data_dir)
```

Comment:
Hello @itsmeakashgoyal ,
I suggestion would be to use python's `subprocess` library to find the location of `llvm-profdata` & `llvm-cov`.
This will remove hard-coded file locations.

Reply:
Thanks @pkthapa.
I think we can create a softlink that these changes not needed. Let me verify at my end.

Reply:
Hello @itsmeakashgoyal ,
Softlinks are created and the docker image is also updated.
PR [here](https://github.com/vmware/concord-bft/pull/1919).